server/www/index.html

```diff
@@ -41,14 +41,13 @@
 
         <div id="bootstrap-app" class="hidden">
 
-            <mode-bar></mode-bar>
+            <mode-bar current-user="currentUser"></mode-bar>
```

Comment:
I'm not really clear why this works while the previous method didn't but it works